contracts/modules/licensing/LicensingModule.sol

```diff
@@ -284,6 +284,11 @@ contract LicensingModule is
         address childIpId,
         bytes calldata royaltyContext
     ) external nonReentrant verifyPermission(childIpId) {
+        LicensingModuleStorage storage $ = _getLicensingModuleStorage();
+        if ($.ipIdParents[childIpId].length() > 0) {
```

Comment:
Is there a reason why we want to enforce Ips to only link to parents once at the licensing module level?

This is originally a royalty-related restriction of using absolute percentages and is also enforced there. 

If in the future a new royalty policy allows to link to parents multiple times somehow - the licensing will not allow it.

Is this something we want to permanently not allow? (linking Ip to parents more than once?)